config.py

```diff
@@ -10,6 +10,7 @@
 class Config(object):
     DEBUG = False
     TESTING = False
+    DEVELOPMENT = False
```

Comment:
I think it should remain True as the default behavior of the system (as it's better to be secured by default and skip authorization only in special cases - like development - in my opinion).

Reply:
If I'll set it to true in this base config class, the default behavior would be to skip validation. 
By setting it to false, the validation will be requried. 
Only in cases where Development = True it will skip the validation. 
I will add DEVELOPMENT = True to the testing config.